yt_dlp/extractor/epoch.py

```diff
@@ -0,0 +1,61 @@
+from .common import InfoExtractor
+from ..utils import (
+    unescapeHTML
+)
+
+
+class EpochIE(InfoExtractor):
+    _VALID_URL = r'https?://www.theepochtimes\.com/[\w-]+_(?P<id>\d+).html'
+    _TESTS = [
+        {
+            'url': 'https://www.theepochtimes.com/they-can-do-audio-video-physical-surveillance-on-you-24h-365d-a-year-rex-lee-on-intrusive-apps_4661688.html',
+            'info_dict': {
+                "id": '4661688',
+                'ext': 'mp4',
+                'url': 'http://vs1.youmaker.com/assets/a3dd732c-4750-4bc8-8156-69180668bda1/playlist.m3u8',
+                'title': '‘They Can Do Audio, Video, Physical Surveillance on You 24H/365D a Year’: Rex Lee on Intrusive Apps',
+                'webpage_url_domain': 'theepochtimes.com',
+                'extractor': 'Epoch',
+                'extractor_key': 'Epoch'
```

Comment:
```suggestion
                'title': '‘They Can Do Audio, Video, Physical Surveillance on You 24H/365D a Year’: Rex Lee on Intrusive Apps',
```
Same below

Reply:
sorry i'm a little confused. do you mean fields `url`, `webpage_url_domain`, `extractor`, `extractor_key` aren't necessary?

Reply:
Yes, they shouldn't be used in tests